cmd/divide.go

```diff
@@ -0,0 +1,272 @@
+// Copyright (c) Bruno Schaatsbergen
+// SPDX-License-Identifier: MIT
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"math/big"
+	"net"
+	"strconv"
+
+	"github.com/bschaatsbergen/cidr/pkg/core"
+	"github.com/bschaatsbergen/cidr/pkg/helper"
+	"github.com/spf13/cobra"
+)
+
+type DivideOptions struct {
+	Network *net.IPNet
+	Divisor int64
+	Users   []int64
```

Comment:
If possible, let's rename "Users' to "Hosts' in this file? It's a more common accepted term that folks grasp easier.